src/boost_histogram/_internal/hist.py

```diff
@@ -1071,7 +1071,7 @@ def variances(self, flow: bool = False) -> Optional["np.typing.NDArray[Any]"]:
                 view.variance,  # type: ignore[union-attr]
                 view.sum_of_weights,  # type: ignore[union-attr]
                 out=np.full(view.sum_of_weights.shape, np.nan),  # type: ignore[union-attr]
-                where=view.sum_of_weights > 1,  # type: ignore[union-attr]
+                where=view.sum_of_weights**2 > view.sum_of_weights_squared,
```

Comment:
What the... Black, this comment was needed!

Reply:
Opened an issue at https://github.com/psf/black/issues/2873, and worked around this by moving this subexpression to a variable, which is a bit more readable, too - but it should not have deleted this comment!